test/e2e/step-definitions/office-steps.ts

```diff
@@ -17,17 +17,13 @@ Before({ tags: '@catc' }, () => {
   this.testCategory = 'c';
 });
 
-Before({ tags: '@catce' }, () => {
-  this.testCategory = 'c';
-});
-
 Before({ tags: '@catc1' }, () => {
   this.testCategory = 'c';
 });
 
 When('I complete the office write up', () => {
   enterRouteNumber('2');
-  if (this.testCategory === 'be' || this.testCategory === 'c') {
+  if (this.testCategory === 'be' || this.testCategory === 'c' || this.testCategory === 'c1') {
```

Comment:
Not sure if this would help but on the dev side we TestCategory.C or TestCategory.C1 to try and avoid using magic strings. It may require some rework to use as categories are normally upper case.